doc/classes/BakedLightmap.xml

```diff
@@ -32,6 +32,7 @@
 			Grid subdivision size for lightmapper calculation. The default value will work for most cases. Increase for better lighting on small details or if your scene is very large.
 		</member>
 		<member name="bake_default_texels_per_unit" type="float" setter="set_bake_default_texels_per_unit" getter="get_bake_default_texels_per_unit" default="20.0">
+			If a [code]lightmap_size_hint[/code] isn't specified on a [code]Mesh[/code] then the lightmap baker will dynamically set the lightmap size using this value. This value is measured in texels per world unit. The maximum lightmap texture size is clamped at 4096x4096.
```

Comment:
```suggestion
			If a [code]lightmap_size_hint[/code] isn't specified on a [code]Mesh[/code], the lightmap baker will dynamically set the lightmap size using this value. This value is measured in texels per world unit. The maximum lightmap texture size is 4096×4096.
```

Reply:
I like that change so I'll change it to that. I'm going to go ahead and fill out the rest of the docs for the BakedLightmap node as well while I'm at it.